backends/advanced/src/advanced_omi_backend/main.py

```
@@ -320,6 +323,25 @@ async def lifespan(app: FastAPI):
     processor_manager = init_processor_manager(CHUNK_DIR, ac_repository)
     await processor_manager.start()
 
+    application_logger.info("Application-level processors started")
+    
+    # Initialize simple queue system
+    try:
+        queue = await get_simple_queue()
+        application_logger.info("Simple queue system started")
+    except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)